code/modules/virus2/effect/wordlists.dm

```diff
@@ -0,0 +1,71 @@
+//For lists of words to be used with viruses.
+var/global/list/virus_opposite_word_list
+
+/datum/disease2/proc/initialize_word_lists()
```

Comment:
I was accounting for the fact that someone else might at some point make a symptom that needs to use one or more such word lists, so I made it a basic disease-level proc so any symptom could call it.